README.md

```diff
@@ -572,5 +572,30 @@ type Account struct {
 }
 ```
 
+### Add extension info to struct field
+
+```go
+type Account struct {
+    ID   int    `json:"id"   extensions:"x-nullable,x-abc=def"` // extensions fields must start with "x-"
+}
+```
+
+generate swagger doc as follows:
+
+```go
+"Account": {
+    "type": "object",
+    "properties": {
+        "id": {
+            "type": "string",
+            "x-nullable": true,
+            "x-abc": "def"
+        }
+    }
+}
+```
+
+Maybe some other tools will use extension info. For example, [go-swagger/go-swagger](https://github.com/go-swagger/go-swagger) use `x-nullable` to render struct field as pointer when generate client. [reference](https://goswagger.io/use/models/schemas.html)
```

Comment:
@lutaoact can you delete this paragraph? 

> Maybe some other tools will use extension info. For example, [go-swagger/go-swagger]

Reply:
@easonlin404 ok, done.